### PR TITLE
Send certificate to node after dissemination + tests

### PR DESCRIPTION
**Pull request by zeegomo**

### CODE REVIEW

File: nomos-cli/src/cmds/disseminate/mod.rs [hunk 1]
```diff
@@ -185,7 +196,31 @@ impl ServiceCore for DisseminateService {
...
+                                .await?;
+                            if !res.status().is_success() {
+                                tracing::error!(
+                                    "Failed to send certificate to node: {}",
+                                    res.status()
+                                );
+                                std::process::exit(1);
+                            }
+                            terminal_cmd_done();
+                        }
```
Comment (danielSanchezQ) [inline nomos-cli/src/cmds/disseminate/mod.rs:222]: Nitpick: what if both options are activated? 🤔

**Review by danielSanchezQ — APPROVED** [review]
Looking good. ~~Left a comment on usability but it can be checked later if needed.~~

File: nomos-cli/src/cmds/disseminate/mod.rs (hunk 1 above)
Comment (zeegomo) [reply nomos-cli/src/cmds/disseminate/mod.rs:222]: we do both?

Comment (danielSanchezQ) [reply nomos-cli/src/cmds/disseminate/mod.rs:222]: Yep my bad. Had the impresion that `terminal_cmd_done` finalized the process!!

**Comment by zeegomo** [comment]
I really don't understand what's happening with the tests :(

**Comment by bacv** [comment]
> I really don't understand what's happening with the tests :(
Seems like an infinite loop somewhere, workflows will be terminated after 6 hours by github.

**Comment by danielSanchezQ** [comment]
> > I really don't understand what's happening with the tests :(
> 
> Seems like an infinite loop somewhere, workflows will be terminated after 6 hours by github.

**6h**? 💀

**Review by youngjoon-lee — APPROVED** [review]

**Pull request merged**